source/proposals/writing-testable-tasks

```diff
@@ -0,0 +1,17 @@
+Writing Testable Bolt Tasks and Plans
+-------------------------------------
+
+Note: I think this might be a good 5-10 minutes lightning/ignite talk
+
+Short Description
+-----------------
+
+Long Description
+----------------
+
+Bolt is great at automating ad-hoc tasks, but how do you know if your tasks
```

Comment:
Call out what it means to test plans as well? Maybe show off helpers in bolt_spec/plans.